libr/asm/p/asm_x86_nz.c

```diff
@@ -2466,6 +2466,10 @@ static int oppush(RAsm *a, ut8 *data, const Opcode *op) {
 			}
 		}
 	} else {
+		if (a->bits == 32 && (op->operands[0].immediate >> 32) > 0) {
```

Comment:
I think is better to drop the >0. Negative values could be a problem if imm is signed i think

```suggestion
		if (a->bits == 32 && (op->operands[0].immediate >> 32)) {
```

Reply:
^